client/post-editor/media-modal/detail/detail-item.jsx

```diff
@@ -32,38 +30,28 @@ class EditorMediaModalDetailItem extends Component {
 		hasNextItem: PropTypes.bool,
 		onShowPreviousItem: PropTypes.func,
 		onShowNextItem: PropTypes.func,
-		onEdit: PropTypes.func
+		onEdit: PropTypes.func,
+		onRestore: PropTypes.func,
 	};
 
 	static defaultProps = {
 		hasPreviousItem: false,
 		hasNextItem: false,
 		onShowPreviousItem: noop,
 		onShowNextItem: noop,
-		onEdit: noop
+		onEdit: noop,
+		onRestore: noop,
 	};
 
-	renderEditButton( className ) {
+	renderEditButton() {
 		const {
 			item,
 			onEdit,
 			site,
 			translate
 		} = this.props;
 
-		// Do not render edit button for private sites
-		if ( site.is_private ) {
-			debug( 'Do not show `Edit button` for private sites' );
-
-			return null;
-		}
-
-		if (
-			! config.isEnabled( 'post-editor/image-editor' ) ||
-			! userCan( 'upload_files', site ) ||
-			isJetpack( site ) ||
-			! item
-		) {
+		if ( ! userCan( 'upload_files', site ) ) {
```

Comment:
Are we enabling the image editor for prod here? We should still check whether the `post-editor/image-editor` is enabled in config.